telegram/ext/_updater.py

```diff
@@ -53,6 +53,7 @@
 
 
 _UpdaterType = TypeVar("_UpdaterType", bound="Updater")  # pylint: disable=invalid-name
+_DefaultIP = DefaultValue("127.0.0.1")
```

Comment:
If this is supposed to be used outside of this module, it should not start with an underscore. TBH I personally would move this to `defaultvalue.py` b/c that's where we define the other singletons, but that's not a deabreaker for me :D

Reply:
no I want to move this